pyproject.toml

```
@@ -125,3 +126,7 @@ files = "./python/constellation/core"
 exclude = [
     "^test_",
 ]
+
+[tool.isort]
+profile = "black"
+line_length = 125
```

```suggestion
line_length = 125
skip_gitignore = true
```
Should skip git ignore files. Also move this section below black so that we have formatting options all after each other